CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [[Unreleased]]
 
+### Fixed
```

Comment:
This should be in Added :)

Reply:
I disagree - it's not really a new feature (aka I wouldn't consider this requiring a minor version bump)

Reply:
I'm ok either way - I don't really think it was broken before adding this, so I tend to think of it as a feature, but also agree that it'd be fine to release this without a minor version bump.

Reply:
Yeah `changed` seems reasonable :)